pbsmetrics/config/metrics_test.go

```diff
@@ -123,8 +147,8 @@ func TestMultiMetricsEngine(t *testing.T) {
 	VerifyMetrics(t, "AdapterMetrics.AppNexus.NoBidMeter", goEngine.AdapterMetrics[openrtb_ext.BidderAppnexus].NoBidMeter.Count(), 5)
 }
 
-func VerifyMetrics(t *testing.T, name string, expected int64, actual int64) {
+func VerifyMetrics(t *testing.T, name string, actual int64, expected int64) {
 	if expected != actual {
-		t.Errorf("Error in metric %s: expected %d, got %d.", name, expected, actual)
+		t.Errorf("Error in metric %s: got %d, expected %d.", name, expected, actual)
```

Comment:
i believe you want this to be: `t.Errorf("Error in metric %s: got %d, expected %d.", name, actual, expected)`

Reply:
Good catch! Corrected